GitUI/UserControls/FileStatusList.Designer.cs

```diff
@@ -57,7 +57,7 @@ private void InitializeComponent()
             FileStatusListView.ShowItemToolTips = true;
             FileStatusListView.Size = new Size(682, 464);
             FileStatusListView.Sorting = SortOrder.Ascending;
-            FileStatusListView.TabIndex = 4;
+            FileStatusListView.TabIndex = 3;
```

Comment:
Are all those changes to this file still relevant?

Reply:
Yes, only FilterBox and the list are tab stops